app/src/main/java/com/myvetpath/myvetpath/CreateSubActivity.java

```diff
@@ -43,4 +167,130 @@ public void onClick(View view) {
+        c.set(Calendar.YEAR, year);
+        c.set(Calendar.MONTH, month);
+        c.set(Calendar.DAY_OF_MONTH, dayOfMonth);
+
+        if(selectedCalendar == SAMPLE_COLLECTED_DATE){
+            String currentDateString = "Collected On " + DateFormat.getDateInstance(DateFormat.FULL).format(c.getTime());
+            TextView dateSelectedTV = (TextView) findViewById(R.id.CollectionDateTV);
+            dateSelectedTV.setText(currentDateString);
+            collectionDate = new GregorianCalendar(year, month, dayOfMonth).getTime();
+        }else if (selectedCalendar == BIRTH_DATE){
+            String currentDateString = "Animal Born On " + DateFormat.getDateInstance(DateFormat.FULL).format(c.getTime());
+            TextView dateSelectedTV = (TextView) findViewById(R.id.Birth_Date_Message_TV);
+            dateSelectedTV.setText(currentDateString);
+            birthDate = new GregorianCalendar(year, month, dayOfMonth).getTime();
+        }else if (selectedCalendar == DEATH_DATE){
+            String currentDateString = "Animal Died On " + DateFormat.getDateInstance(DateFormat.FULL).format(c.getTime());
+            TextView dateSelectedTV = (TextView) findViewById(R.id.Death_Date_Message_TV);
+            dateSelectedTV.setText(currentDateString);
+            deathDate = new GregorianCalendar(year, month, dayOfMonth).getTime();
+        }
+
+
+    }
+
+    //The selectedSex value is set whenever the user changes the sex in the spinner menu.
+    @Override
+    public void onItemSelected(AdapterView<?> adapterView, View view, int i, long l) {
+        selectedSex = adapterView.getItemAtPosition(i).toString();
+    }
+
+    //This function is needed to implement the spinner interface, but it will probably never be called
+    @Override
+    public void onNothingSelected(AdapterView<?> adapterView) {
+
+    }
+
+    //This is called whenever the euthanized checkbox is clicked. Changes the data that will be stored in teh database
+    public void onCheckboxClicked(View view){
+        //Is the view now checked?
+        boolean checked = ((CheckBox) view).isChecked();
+        //Check which checkbox was clicked
+
+        switch(view.getId()){
+            case R.id.EuthanizedCB:
+                isEuthanized = checked;
+                break;
+        }
+
+
+    }
+
+    // Purpose: check if string is an integer. This will be used to validate that the user entered an integer into a numeric field
+    private boolean isInteger(String str) {
+        try
+        {
+            Integer.parseInt(str);
+            return true;
+        }
+        catch (NumberFormatException e)
+        {
+            return false;
+        }
+    }
+
+    //This method stores all the data into the database. Called whenever the user wants to save a submission
+    //Todo: store everything into database. May need to make changes to this to accomodate differences between submit button and save as draft button
+    private void storeDataInDB(){
```

Comment:
Is this still in the demo stage? I don't see where the information is being sent to the database. I only see the information being prepared.

Reply:
Yes, we'll write the data into the database once those tables are set up. This is mostly just a placeholder for now.